packages/common/browser/leaders-company-social-link.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <a
+    :href="href"
+    :target="target"
+    class="social-icon-link"
+    :title="title"
+    :rel="rel"
+    @click="emitAction"
+  >
+    <component :is="icon" :modifiers="modifiers" />
+  </a>
+</template>
+
+<script>
+import Youtube from '@base-cms/marko-web-icons/browser/youtube.vue';
+import Linkedin from '@base-cms/marko-web-icons/browser/linkedin.vue';
+import Facebook from '@base-cms/marko-web-icons/browser/facebook.vue';
+import Twitter from '@base-cms/marko-web-icons/browser/twitter.vue';
+
+export default {
+  components: {
+    'icon-youtube': Youtube,
```

Comment:
Change your imports above and just set directly... 

`import IconYoutube from '@base-cms/marko-web-icons/browser/youtube.vue';`

```js
components: {
  IconYoutube,
}
```